tsl/src/continuous_aggs/invalidation_threshold.c

```diff
@@ -284,3 +251,63 @@ invalidation_threshold_compute(const ContinuousAgg *cagg, const InternalTimeRang
 
 	return refresh_window->end;
 }
+
+void
+invalidation_threshold_insert(const ContinuousAgg *cagg, int64 threshold, bool threshold_isnull)
```

Comment:
It is a little bit confusing that setting `threshold_isnull` to `true`, which is the case in the one place this function is called, actually doesn't mean we insert a NULL value. Instead, it means we insert a MIN value, but this is only obvious from reading the code.

It makes more sense to me to either insert the NULL value to mean "initial" state, or we don't allow inserting NULL values at all and in that case ensure we call the function with threshold=MIN.

The other option is to just rename the function to `invalidation_threshold_initialize()` and then we don't pass any threshold at all and internally set the MIN. 

I would actually go with the last suggestion (initialize name) because the function is only called once anyway, specifically to initialize the threshold.

Reply:
Ohhh... this was a leftover of the initial implementation, make a lot of sense make it simple renaming it to `invalidation_threshold_initialize`